app/views/waste_calendar/create.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="d-sm-flex align-items-center justify-content-between">
+  <h1 class="h3 mb-0 text-gray-800">Schritt 2: Daten für Import zuweisen</h1>
+</div>
+
+<p>Die Spalten aus beiden CSV-Quellen werden nun den Feldern aus dem Abfallkalender zugewiesen.</p>
+
+<div class="row">
+  <div class="col-lg-8">
+    <%= form_with( url: "/waste_calendar", method: :post, local: true) do |f| %>
+      <%= hidden_field_tag :address_data, @address_data %>
+      <%= hidden_field_tag :tour_data, @tour_data %>
+      <div class="row">
+        <div class="col">
+          <h2 class="d-sm-flex align-items-center justify-content-between my-4">
+            Adressdaten
+          </h2>
+          <div class="form-group">
+            <label for="description">Straße:</label>
+            <div> <%= select_tag("address[street]", options_for_select(@address_data_selects), include_blank: true, class: "form-control" ) %></div>
+          </div>
+          <div class="form-group">
+            <label for="description">PLZ:</label>
+            <div> <%= select_tag("address[zip]", options_for_select(@address_data_selects), include_blank: true, class: "form-control" ) %></div>
```

Comment:
manche selects sind in <div> gewrappt, andere nicht. braucht es insgesamt nicht. nehme die <div>s um selects alle raus.

Reply:
done in 8def4ab5bbc68a9c690785024135d820842578c3